### ISSUE TITLE
Interaction with window.scrollTo/scrollBy

### ISSUE THREAD

**Issue by staktrace** [WICG/visual-viewport#12]
I think the interaction of the visual viewport with the window.scrollTo/scrollBy is also worth specifying as part of this proposal. The intuitive (to me, anyway) behaviour from the author's point of view would be that calling window.scrollTo also resets the visual viewport to the top-left of the layout viewport, and that calling window.scrollBy leaves the visual viewport as-is relative to the layout viewport.

**Comment by bokand** [comment]
Agreed that this needs some thought but I'm not sure yet how these should work. My intuition is that if we add visualViewport.scrollTo/By then the old `window` versions should behave as if the visual viewport didn't exist. That's not the case for us today.
